### ISSUE TITLE
The extension is asking me to manually save a file every time it needs to create one. It had been working fine until yesterday.

### ISSUE THREAD

**Issue by samuel-mr** [microsoft/vscode-copilot-release#3574]
Type: <b>Bug</b>

Just enter every prompt that need to create a new file in the code base

Extension version: 0.24.2024121201
VS Code version: Code 1.96.2 (Universal) (fabdb6a30b49f79a7aba0f2ad9df9b399473380f, 2024-12-19T10:22:47.216Z)
OS version: Darwin arm64 24.1.0
Modes:

<details><summary>Prompt Details</summary>
<pre>
Info: Using the Electron fetcher.
Info: Initializing Git extension service.
Info: Logged in as samuel-mr
Info: Successfully activated the vscode.git extension.
Info: Enablement state of the vscode.git extension: true.
Info: Successfully registered Git commit message provider.
Debug: [context keys] Updating context keys.
</pre>
</details><details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|2, 2, 2|
|Memory (System)|8.00GB (0.07GB free)|
|Process Argv|. --crash-reporter-id 2ecd5e8d-33af-4512-8b86-158be8e94718|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
vscrpc:30673769
2i9eh265:30646982
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupytercf:31046870
2f103344:31071589
nativerepl1:31139838
pythonrstrctxt:31112756
nativeloc1:31192215
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530
6074i472:31201624

```

</details>

**Comment by samuel-mr** [comment]
![Image](https://github.com/user-attachments/assets/bcc137dd-a5b7-42b1-a6f3-c82c92d87d71)
This is the window that shows me for each file, I have already restarted the app, I have reinstalled the app, I still have the problem

**Comment by doggy8088** [comment]
@samuel-mr  Yes, I occasionally encounter this issue myself. Often, pressing Discard and then re-editing will resolve it. I think this is an issue that must be fixed.
